cmd/kubelet/app/server.go

```diff
@@ -711,13 +711,11 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 		}
 
 		var topologyManagerPolicyOptions map[string]string
-		if utilfeature.DefaultFeatureGate.Enabled(features.TopologyManager) {
-			if utilfeature.DefaultFeatureGate.Enabled(features.TopologyManagerPolicyOptions) {
-				topologyManagerPolicyOptions = s.TopologyManagerPolicyOptions
-			} else if s.TopologyManagerPolicyOptions != nil {
-				return fmt.Errorf("topology manager policy options %v require feature gates %q, %q enabled",
-					s.TopologyManagerPolicyOptions, features.TopologyManager, features.TopologyManagerPolicyOptions)
-			}
+		if utilfeature.DefaultFeatureGate.Enabled(features.TopologyManagerPolicyOptions) {
+			topologyManagerPolicyOptions = s.TopologyManagerPolicyOptions
+		} else if s.TopologyManagerPolicyOptions != nil {
+			return fmt.Errorf("topology manager policy options %v require feature gates %q, %q enabled",
+				s.TopologyManagerPolicyOptions, features.TopologyManager, features.TopologyManagerPolicyOptions)
```

Comment:
why it still refers to `features.TopologyManager`?

```suggestion
			return fmt.Errorf("topology manager policy options %v require feature gate  %q enabled",
				s.TopologyManagerPolicyOptions, features.TopologyManagerPolicyOptions)
```

Reply:
Good point, will remove.